core.json

```diff
@@ -1103,9 +1107,27 @@
               "format":"int"
             }
           }
+        },
+        {
+          "name": "Old time",
```

Comment:
Might be picky, but couldn't _"Previous time"_ express this better? As it could potentially ber rescheduled multiple times only the previous time is stored.

Reply:
Yeah, but the fact that it could be rescheduled multiple times was the reason I didn't go with a more specific "Previous time"/"Original time". That would mean that we commit to only storing **one** previous time, either the last one, or the original. So I landed on providing an `updated` timestamp in the data attributes so that it's possible to see when the time had changed... so maybe that should be `changed`...